src/extensions/default/InlineColorEditor/main.js

```diff
@@ -27,8 +27,26 @@ define(function (require, exports, module) {
     var EditorManager       = brackets.getModule("editor/EditorManager"),
         ExtensionUtils      = brackets.getModule("utils/ExtensionUtils"),
         InlineColorEditor   = require("InlineColorEditor").InlineColorEditor,
-        ColorUtils          = brackets.getModule("utils/ColorUtils");
+        ColorUtils          = brackets.getModule("utils/ColorUtils"),
+        properties          = JSON.parse(require("text!ColorProperties.json"));
 
+    var DEFAULT_COLOR = "white";
+
+    /*
+    This function takes a line number, string and the colon position, and writes
+    the default color value at 2 places after the colon position
+     */
+    function setLine(lineNo, newContent, hostEditor, colonPos) {
```

Comment:
You can find the docs here: https://codemirror.net/doc/manual.html#api_content. If you Ctrl+F `replaceRange`, you should be able to find it in a bunch of places (esp. in Editor.js)